tests/test_draft_service.py

```diff
@@ -53,6 +57,56 @@ def test_create_draft_of_existing_record(app, draft_service, record_service,
         assert identified_draft.record[key] == value
 
     # Check the actual record was not modified
-    identified_record = record_service.read(id_=recid, identity=fake_identity)
+    identified_record = draft_service.read(id_=recid, identity=fake_identity)
```

Comment:
[Discussion] Why not keep using record_service here? 

I know @lnielsen mentioned something related to that... Seems we are giving 2 responsibilities to RecordDraftService: servicing Records AND servicing Drafts. I don't understand why :smile:  . RecordService is not going away. It can still be accessed. So we have 2 things that are responsible for servicing Records...

Reply:
resolved IVL (in video life - it's going to catch on I tell you ;) )

Reply:
Discussed IRL. Conclusion: Naming problem to be seen according to other users acceptance/confusion. "issue" shelved. (Applies to other points on this PR).